ts/components/screens/PinCreation/PinCreation.tsx

```diff
@@ -223,10 +224,20 @@ export const PinCreation = ({ isOnboarding = false }: Props) => {
     <View testID="pin-creation-screen" style={IOStyles.flex}>
       <View style={[IOStyles.flex, IOStyles.centerJustified]}>
         <VSpacer size={8} />
-        <View style={IOStyles.alignCenter}>
-          <Pictogram name="key" size={64} />
-        </View>
-        <VSpacer size={8} />
+        {/*
+          if the device height is > 780 and the OS isn't
+          Android, then the pictogram will be visible,
+          otherwise it will not be visible
```

Comment:
```suggestion
          If the device height is less than MIN_HEIGHT_TO_SHOW_FULL_RENDER and the OS is
          Android, then the pictogram will not be visible.
          Otherwise, the pictogram will be visible.
```